build.gradle

```diff
@@ -481,7 +496,7 @@ task gatkDoc(type: Javadoc, dependsOn: classes) {
             into gatkDocDir
         }
     }
-    source = sourceSets.main.allJava
+    source = sourceSets.main.allJava + getPicardJavaSourceFiles(picardVersion)
```

Comment:
I guess that `getPicardJavaSourceFiles` can be removed and being simplify the gatkDoc/gatkTabComplete by using:

```gradle
 final docSources = configurations.docgen.collect { zipTree(it) }
source = files(docSources) + sourceSets.main.allJava
include '**/*.java'
```

This will also help to include new dependencies in the code and not only Picard....

Reply:
@magicDGS - thanks - I'll try that. Although I don't imagine we'll add any more source sets like this, your suggestion seems cleaner all around.